modules/platforms/cpp/core/src/impl/cluster/cluster_group_impl.cpp

```diff
@@ -573,6 +573,10 @@ namespace ignite
                 return topVer;
             }
 
+            // The empty cluster group could be created using ForNodeId()
+            // method with not exist (Guid(0, 0)) Cluster Node Id.
+            // It is required for ForPredicate() implementation
+            // to avoid situation when two ClusterGroupImpl's uses same jobject.
```

Comment:
This comment should be moved to function body.

Reply:
Fixed.